checkST2110.js

```diff
@@ -454,6 +458,30 @@ const test_10_62_1 = (sdp, params) => {
   return errors;
 };
 
+const tsmodePermitted2022 = ['SAMP', 'NEW', 'PRES'];
+
+// ST 2110-10:2022 Section 8.7 Test 1 - RTP Timestamp Mode and Delay, TSMODE and TSDELAY parameters are an acceptable value.
+const test_10_87_1 = (sdp, params) => {
```

Comment:
Hm, now we're mixing function names based on section numbers in different revisions of the spec...

Reply:
Yeah, it's confusing - unless we break the convention and call it `test_10_2022_87_1` or something?

Reply:
Yep, seems like a good idea?